pyvera/subscribe.py

```diff
@@ -132,6 +132,10 @@ def _run_poll_server(self):
                 LOG.info("Could not contact Vera - will retry in %ss",
                          SUBSCRIPTION_RETRY)
                 time.sleep(SUBSCRIPTION_RETRY)
+            except json.decoder.JSONDecodeError:
+                LOG.exception("Response was malfomed - will retry in %ss",
```

Comment:
Probably makes sense just to log as `info` rather than `exception` - since we're retrying.